client/package.json

```diff
@@ -13,7 +13,11 @@
     "lint": "eslint src"
   },
   "dependencies": {
-    "apollo-client": "^1.9.3",
+    "apollo-cache-inmemory": "^0.2.0-rc.4",
+    "apollo-client": "^2.0.0-rc.7",
```

Comment:
The web app is using "apollo-client": "^2.2.3" - any reason you are using an older version?

Reply:
guide I followed used this version